resources/aws_ec2.go

```diff
@@ -175,6 +185,17 @@ func (obj *AwsEc2Res) Init() error {
 	obj.closeChan = make(chan struct{})
 	obj.wg = &sync.WaitGroup{}
 
+	if obj.WatchListenAddr != "" {
```

Comment:
Does any of this have a matching close or cleanup part it should do that we need to do in `Close()` ?